WidgetKit/OpenTabs/OpenTabsWidget.swift

```diff
@@ -0,0 +1,177 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import SwiftUI
+import WidgetKit
+import UIKit
+import Combine
+
+struct TabProvider: TimelineProvider {
```

Comment:
Suggestion: Wondering if we can move some business logic ouf of this file to either its own file. Ex. timelines can and logic in one file and view in other along with view representation. 

What do you think of this @sblatz ?

Reply:
Done!